src/Database/SqlServer/Generator.hs

```diff
@@ -46,6 +46,14 @@ data GenerateOptions = GenerateOptions
   , excludeTypes :: RenderOptions
   }
 
+defaultGenerateOptions :: GenerateOptions
```

Comment:
This looks great.

The only constructive feedback I can give is around the use of `undefined`.  By leaving these as `undefined`, you've got no way of doing anything nice with them.  You can't evaluate `undefined` or compare anything to is, it's a bottom type.

Would it be possible to give a default `size` and `seed`?